### ISSUE TITLE
Theme support (FilePicker)

### ISSUE THREAD

**Issue by KillerBLS** [gzu-liyujiang/AndroidPicker#138]
### Sorry for my English, if there is a mistakes in the text

Could you provide support for Light/Dark/Material themes (with icons) in the next version?
Something like this:

```java
FilePicker picker = ......;
picker.setTheme(Theme.MATERIAL_DARK); //or something like this
picker.setFolderIcon(getResources().getDrawable(R.drawable.ic_folder));
```

or global theming

```xml
<style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
    <item name="picker_file_icon">@drawable/ic_file</item>
    <item name="picker_folder_icon">@drawable/ic_folder</item>
</style>
```

**Comment by liyujiang-gzu** [comment]
Good idea，thanks for your issue

**Comment by liyujiang-gzu** [comment]
```java
picker.setFileIcon(getResources().getDrawable(R.drawable.ic_file));
picker.setFolderIcon(getResources().getDrawable(R.drawable.ic_folder));
picker.setFolderIcon(getResources().getDrawable(R.drawable.ic_folder));
picker.setHomeIcon(getResources().getDrawable(R.drawable.ic_home));
picker.setUpIcon(getResources().getDrawable(R.drawable.ic_updir));
```

**Comment by abiemann** [comment]
how to change the text color of the filenames ?
Right now the filepicker is showing a white background and the text is white

**Comment by liyujiang-gzu** [comment]
@abiemann 
> how to change the text color of the filenames ?
> Right now the filepicker is showing a white background and the text is white.
> 
> NOTE: `picker.setBackgroundColor(Color.BLACK);` has no effect; the background is still white !

Because this:
```java
    protected LinearLayout makeCenterView() {
        LinearLayout rootLayout = new LinearLayout(activity);
...
        rootLayout.setBackgroundColor(Color.WHITE);
        ListView listView = new ListView(activity);
        listView.setBackgroundColor(Color.WHITE);
...
        emptyView = new TextView(activity);
...
        emptyView.setTextColor(Color.BLACK);
...
        return rootLayout;
    }
```
So you can:
```java
picker.show();
changeBgColor(picker.getRootView());

    private void changeBgColor(View view) {
        if (view instanceof ViewGroup) {
            ViewGroup vg = (ViewGroup) view;
            for (int i = 0, n = vg.getChildCount(); i < n; i++) {
                changeBgColor(vg.getChildAt(i));
            }
        } else {
            view.setBackgroundColor(Color.BLACK);
        }
    }
```

**Comment by abiemann** [comment]
Ok, I'll make a note of cycling through views like that. It's my least favorite approach.

I did solve it by `setTheme(android.R.style.Theme_Material_Light_NoActionBar);` within onCreate() of my Activity.

By the way, I did try :

`            FilePicker picker = new FilePicker(this, FilePicker.FILE);
            ...
            picker.setBackgroundColor(Color.BLACK);
            picker.show();`

However, the background was still white - should I write a ticket about it ?
